Sprint-3/quote-generator/index.html

```diff
@@ -3,13 +3,24 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Quote Generator App</title>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Italiana&family=Montserrat+Alternates:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
     <script defer src="quotes.js"></script>
   </head>
   <body>
-    <h1>hello there</h1>
-    <p id="quote"></p>
-    <p id="author"></p>
-    <button type="button" id="new-quote">New quote</button>
-  </body>
+ <main>
+  <div class="container-quotes">
+    <h1>DAILY INSPIRATION</h1>
+      <div class="container">
+        <div>
+          <p id="quote">"Life isn't about getting and having, it's about giving and being."</p>
```

Comment:
Instead of always showing the same quote on page load, you can use JS to set a random quote ** when the page is fully loaded**. (Hint: Find out what event is triggered when a page is fully loaded)

Reply:
Fixed:

I asked chat GTP to explain me, and it said:

 " you can call the same logic for generating a random quote when the page is fully loaded. Use the DOMContentLoaded event to trigger this functionality."
 

New code loading with a new cote. ;) 
I didn't write it was chat GTP, but I got what the changes need.

// Display a random quote on the page load
window.addEventListener("DOMContentLoaded", displayRandomQuote);

______________________________________
function displayRandomQuote() {
  const randomIndex = Math.floor(Math.random() * quotes.length);
  const randomQuote = quotes[randomIndex];

  quoteElement.textContent = randomQuote.quote;
  authorElement.textContent = "- " + randomQuote.author;
}

// Display a random quote on page load
window.addEventListener("DOMContentLoaded", displayRandomQuote);

// Change quote when the button is clicked
btn.addEventListener("click", displayRandomQuote);

_________


thank you ,